dev/jobs/prep.sh

```diff
@@ -28,30 +28,32 @@ GDUMP="gdas"
 
 export OPREFIX="${RUN_local}.t${cyc}z."
 
-YMD=${PDY} HH=${cyc} DUMP=${RUN_local} declare_from_tmpl -rx \
+RUN=${RUN_local} YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
     COMIN_OBS:COM_OBS_TMPL \
     COMOUT_OBS:COM_OBS_TMPL \
-    COMIN_OBSDMP:COM_OBSDMP_TMPL
+    COMINobsproc:COM_OBSPROC_TMPL \
+    COMIN_TCVITAL:COM_TCVITAL_TMPL
 
-RUN=${GDUMP} DUMP=${GDUMP} YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
-    COMIN_OBS_PREV:COM_OBS_TMPL \
-    COMIN_OBSDMP_PREV:COM_OBSDMP_TMPL
+RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
+    COMOUT_OBS_PREV:COM_OBS_TMPL \
+    COMINobsproc_PREV:COM_OBSPROC_TMPL
 
-export MAKE_PREPBUFR=${MAKE_PREPBUFR:-"YES"}
-if [[ ! -d "${COMOUT_OBS}" ]]; then mkdir -p "${COMOUT_OBS}"; fi
+if [[ ! -d "${COMOUT_OBS}" ]]; then
+   mkdir -p "${COMOUT_OBS}"
+fi
```

Comment:
`mkdir -p` does nothing if the directory already exists, so you don't need to wrap this (Note that this is not true if `-p` is not provided):

```suggestion
mkdir -p "${COMOUT_OBS}"
```

Reply:
Thanks @DavidHuber-NOAA !